string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,39 @@
+/* exported titleCase */
+
+function titleCase(title) {
+
+  var i = 0;
+  var singleWord = '';
+  var fullString = '';
+
+  title = title.toLowerCase();
+  title += ' ';
+
+  for (i; i < title.length; i++) {
+
+    while (title[i] !== ' ') {
+
+      if (i === 0 || title[i - 1] === ' ' || singleWord === 'Java' || singleWord === ' Java' || title[i - 1] === '-') {
+        singleWord += title[i].toUpperCase();
+      } else {
+        singleWord += title[i];
+      }
+      i++;
+    }
+
+    if (singleWord.length <= 4 && i - singleWord.length !== 0 && title[i - (singleWord.length + 1)] !== ':' && singleWord !== ' Web') {
+      singleWord = singleWord.toLowerCase();
+    }
```

Comment:
It would be much better to use a list of "short words". You can put them in an array and look them up with the `.includes()` method.